components/centraldashboard-angular/Makefile

```diff
@@ -0,0 +1,14 @@
+IMG ?= centraldashboard
```

Comment:
nit: Let's use `centraldashboard-angular` here as well, to make sure we use the same value everywhere.

We can revert to `centraldashboard` once we are confident with switching the web apps

Reply:
@kimwnasptd Fixed it and pushed again!